CalculatorClone/CalculatorButtonStyle+Extensions.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CalculatorButtonStyle+Extensions.swift
+//  CalculatorClone
+//
+//  Created by 김남건 on 2022/05/28.
+//
+
+import Foundation
+import SwiftUI
+
+extension CalculatorButtonStyle {
+    var backgroundColor: Color {
+        switch buttonContent {
+        case .reset, .plusMinus, .percent:
+            return Color(UIColor.systemGray2)
+        case .one, .two, .three, .four, .five, .six, .seven, .eight, .nine, .zero, .dot:
+            return Color(UIColor.systemGray6)
+        case .divide, .multiply, .minus, .plus, .equals:
+            return .orange
+        }
+    }
+
+    var foregroundColor: Color {
+        switch buttonContent {
+        case .reset, .plusMinus, .percent:
+            return .black
+        default:
+            return .white
+        }
+    }
+
+    var width: CGFloat {
+        buttonContent == .zero ? 177 : 82.5
+    }
+
+    var horizontalPadding: CGFloat {
+        buttonContent == .zero ? 30 : 0
+    }
+
+    var alignment: Alignment {
+        buttonContent == .zero ? .leading : .center
+    }
+
+    var fontSize: CGFloat {
+        switch buttonContent {
+        case .plusMinus, .divide, .multiply:
+            return 30
+        default:
+            return 40
+        }
+    }
+
+    var weight: Font.Weight {
```

Comment:
```suggestion
    var fontWeight: Font.Weight {
```

어떤 weight인지 알기쉬운 명명이면 좋을 것 같아요 :)

Reply:
수정하였습니다!